apps/frontend/src/app/components/header/header.component.html

```diff
@@ -24,9 +24,17 @@ <h1 class="title" routerLink="/">Verseghy</h1>
         <a (click)="drawer = !drawer" href="" routerLink="/misc/authors">Fejlesztők</a>
       </div>
     </div>
-    <a class="ekreta" href="https://klik035992001.e-kreta.hu/Adminisztracio/Login" target="_blank">
-      <img alt="" src="assets/icons/kreta2.png" />
-    </a>
+    <div (click)="kretaSubmenu = !kretaSubmenu" [ngClass]="{ open: kretaSubmenu }" class="menu ekreta">
+      <img alt="" src="assets/icons/kreta2.png" /><i class="material-icons arrow">expand_more</i>
+      <div class="submenu">
+        <a href="https://klik035992001.e-kreta.hu/Adminisztracio/Login" target="_blank" (click)="drawer = !drawer">
```

Comment:
Since there is a click handler on this a tag the default behaviour (opening the tab in new window) is not triggered always. Please make sure to always `preventDefault()` the click event and open the new window manually.

Reply:
Done in 7d22f75